docs/doxygen/images/retry_utils_flow.pu

```diff
@@ -0,0 +1,18 @@
+@startuml
+
+skinparam dpi 300
+skinparam ArrowFontSize 18
+
+start
+: **RetryUtils_ParamsReset()**;
```

Comment:
A note on the diagram for each of the `RetryUtils_ParamsReset` and `RetryUtils_BackOffAndSleep` functions would be helpful

```suggestion
: **RetryUtils_ParamsReset()**;
note: Initializes retry parameters
```

Reply:
It is already explained in the docs.